site/_includes/explanations/non-persistent-topics.md

```diff
@@ -0,0 +1,57 @@
+As name suggests, non-persist topic does not persist messages into any durable storage disk unlike persistent topic where messages are durably persisted on multiple disks. 
```

Comment:
Can you also add the warning for experimental? Something like  in the download page where we have: 

```
{% include admonition.html type="success" title='Notice' content="
The current release was done prior to entering the Apache Incubator, so
it's still relative to 'Yahoo Pulsar' rather than 'Apache Pulsar'" %}

```